tools/spy-detector/src/main.rs

```diff
@@ -0,0 +1,158 @@
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+use std::time::{Duration, Instant};
+
+use nng::options::protocol::pubsub::Subscribe;
+use nng::options::Options;
+use nng::{Protocol, Socket};
+
+use shared::event_msg;
+use shared::event_msg::event_msg::Event;
+use shared::net_msg;
+use shared::prost::Message;
+use simple_logger::SimpleLogger;
+
+const ADDRESS: &str = "tcp://127.0.0.1:8883";
+const STATS_INTERVAL: Duration = Duration::from_secs(120); // Duration(seconds) to print stats of peers
```

Comment:
we should make these configurable through command line arguments with clap. See #30 for an example

Reply:
Correct!! we can pass some arguments using clap to configure that!!